### PR TITLE
#12390 - Added missing external message fields

### LINKED ISSUE #12390: Add missing fields from eSanté to the ExternalMessage entity [1]
### Feature Description

Several new fields required for eSanté have been added to the ExternalMessageDto. So far, these have not yet been added to the entity class, which is supposed to be done in this issue.

Additionally, captions should be added for all the new external message and test report fields as those are needed for the UI in a follow-up issue.

### Added Value/Benefit

Values transferred from eSanté can actually be stored in the database.

### Acceptance Criteria

- [ ] The `ExternalMessage` entity has been extended by the following fields:
  - `caseReportDate`
  - `personExternalId`
  - `personNationalHealthId`
  - `personPhoneNumberType`
  - `personCountry`
- [ ] Captions have been added for all these fields, as well as for the new prescriber and ct value fields for test reports

### Implementation Details

_No response_

### Mockups

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by MateStrysewske**
Fixes #12390

### CODE REVIEW

**Comment by leventegal-she** [comment]
Please fix the failing test(s)

**Comment by MateStrysewske** [comment]
@leventegal-she Fixed, please check again :)

File: sormas-backend/src/test/java/de/symeda/sormas/backend/externalmessage/ExternalMessageFacadeEjbMappingTest.java [hunk 1]
```diff
@@ -56,6 +60,7 @@ public void testFromDto() {
 		assignee.setUuid("12345");
 
 		when(sampleReportFacade.fromDto(eq(sampleReportDto), any(ExternalMessage.class), eq(false))).thenReturn(sampleReport);
+		when(countryService.getByReferenceDto(source.getPersonCountry())).thenReturn(null);
```
Comment (leventegal-she) [inline sormas-backend/src/test/java/de/symeda/sormas/backend/externalmessage/ExternalMessageFacadeEjbMappingTest.java:63]: would be more useful to mock also the country than assert that the right country has been mapped to dto

**Review by leventegal-she — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12404

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12404

**Pull request merged**